packages/next/src/utilities/createPayloadRequest.ts

```diff
@@ -45,8 +46,10 @@ export const createPayloadRequest = async ({
     language,
   })
 
-  let locale
-  let fallbackLocale
+  const fallbackFromRequest =
+    searchParams.get('fallback-locale') || searchParams.get('fallbackLocale')
```

Comment:
This might be a dumb question, what if I have `?fallbacklocale=` in my request? Is this case-insensitive?